src/content/apis/artificial-intelligence/nlpcloud.md

```diff
@@ -0,0 +1,12 @@
+---
+name: NLPCloud
+description: API for pre-trained or custom models for NER, sentiment-analysis, classification, summarization, paraphrasing, grammar and spelling correction, keywords and keyphrases extraction, chatbot, product description and ad generation, intent classification, text generation, image generation, blog post generation, code generation, question answering, automatic speech recognition, machine translation, language detection, semantic search, semantic similarity, tokenization, POS tagging, embeddings, and dependency parsing.
+categories:
+  - artificial-intelligence
```

Comment:
```suggestion
  - development
```

Reply:
should I rename the category folder to development as well?

Reply:
Oh yes, there already is a development folder, so please move the file accordingly 🙇